coverage.sh

```diff
@@ -7,9 +7,9 @@ for dir in $dirs;
 do
         if ls "$dir"/*.go &> /dev/null;
         then
-            cmdflags="-timeout 20m -v -coverprofile=profile.out -covermode=count $dir"
```

Comment:
Are we using this script anymore?  From the makefile it looks like we are just calling `go test` directly?

Reply:
yeah, travis.yml